src/core/CreateFile.java

```diff
@@ -0,0 +1,47 @@
+package core;
+
+import java.io.IOException;
+import java.nio.file.FileAlreadyExistsException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+/**
+ * Created by Marcelo loayza on 22/05/2016.
+ */
+public class CreateFile {
+
+    private String newFilePath;
+
+    /**
+     * constructor of the class
+     */
+    public CreateFile(String newPath)
+    {
+        newFilePath = newPath;
+
+    }
+    /**
+     * This class create a file using a string for the new file name
+     * and the Path with the location that will be used on the creation
+     */
+
+    public Boolean createNewFile(String filename) throws IOException {
```

Comment:
remember that the documentation should be without empty lines to be associated and needs to follow the next format:
/**
 * Returns an Image object that can then be painted on the screen. 
 * The url argument must specify an absolute {@link URL}. The name
 * argument is a specifier that is relative to the url argument. 
 * <p>
 * This method always returns immediately, whether or not the 
 * image exists. When this applet attempts to draw the image on
 * the screen, the data will be loaded. The graphics primitives 
 * that draw the image will incrementally paint on the screen. 
 *
 * @param  url  an absolute URL giving the base location of the image
 * @param  name the location of the image, relative to the url argument
 * @return      the image at the specified URL
 */
 public Image getImage(URL url, String name) {
        try {
            return getImage(new URL(url, name));
        } catch (MalformedURLException e) {
            return null;
        }
 }